example/docker/amd64-musl-gcc/dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:alpine AS builder
+
+RUN apk update && apk add --no-cache git
+
+RUN git clone https://github.com/enfein/mieru.git /build
+
+WORKDIR /build
+
+RUN go mod tidy
```

Comment:
This step is not needed. The source code already guarantees the modules are tidy.